samples/precompiled/SMSReminder.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
+
+namespace FunctionApp14
+{
+    public static class SMSReminder
+    {
+        [FunctionName("SetReminder")]
+        public static async Task RunOrchestrator(
```

Comment:
Let's add a comment about how this is to be used with the TODO web app sample defined in our samples project with a link to the GitHub page for that sample.